sema/query/__main__.py

```
@@ -252,23 +241,20 @@ def getdelimiter(args: argparse.Namespace):
     The main entry point to this module.
 
     """
-    args = (
-        get_arg_parser().parse_args(sysargs)
-        if sysargs is not None and len(sysargs) > 0
-        else get_arg_parser().parse_args()
-    )
+    args: argparse.Namespace = get_arg_parser().parse_args(cli_args)
+
     enable_logging(args)
     log.info("The args passed to %s are: %s." % (sys.argv[0], args))
```

Ruff

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)